apps/web/pages/cropper.css

```diff
@@ -0,0 +1,58 @@
+.crop-container {
```

Comment:
are you sure there is no way to add this directly in the component with classNames or style object?